pkg/controllers/vdb/verticadb_controller.go

```diff
@@ -200,6 +202,10 @@ func (r *VerticaDBReconciler) constructActors(log logr.Logger, vdb *vapi.Vertica
 		MakeTLSServerCertGenReconciler(r, log, vdb),
 		// Create a ConfigMap to store secret names for all tls certs
 		MakeNMACertConfigMapGenReconciler(r, log, vdb),
+		// rotate https tls cert when tls cert secret name is changed in vdb.spec
+		MakeHTTPSCertRotationReconciler(r, log, vdb, dispatcher, pfacts),
```

Comment:
These should be one of the last reconcilers to run, so we are sure the db is at stable as it could be. I suggest we add them after MakeSaveRestorePointReconciler